### PR TITLE
Modify Readme

### PR DESCRIPTION
**Pull request by nepecg000**
Thanks for reviewing this first pull request.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,3 @@
...
-This repo contains the projects you'll work on throughout SPS.
-
-To get started:
-
-- Login to [Google Cloud Shell](https://ssh.cloud.google.com/cloudshell/editor)
-- Clone this repo: `cd; git clone https://github.com/google/software-product-sprint.git`
-- Open the GitHub setup walkthrough: `teachme ~/software-product-sprint/walkthroughs/week-0-setup/github-setup-walkthrough.md`
-  - If the tutorial panel does not open or display the walkthrough contents, try refreshing the page.
-
-Then follow the on-screen instructions to set up your repo.
```
Comment (layog) [inline README.md:12]: Why did you remove this part?

[NO ACTION REQUIRED] This is a sample comment. A reviewer will add such comments to your code to ask for clarifications or suggest some changes.

**Review by layog — APPROVED** [review]
lgtm.

This is a common abbreviation used to say "Looks good to me". This means that the reviewer has approved your PR (pull request). You can go ahead and merge this PR. Nicely done!

File: README.md (hunk 1 above)
Comment (nepecg000) [reply README.md:12]: Ah. That's how I understood what the instruction said:

## Modify README

To test that everything is connected, modify your
<walkthrough-editor-open-file filePath="software-product-sprint/README.md">README.md</walkthrough-editor-open-file>
file.

This file contains the content that shows in your repo's GitHub page.
Change it to say "This repo contains [your name]'s portfolio and
SPS projects."

The `README.md` file now belongs to you. You should feel free to customize it
and make it your own!
